reports.js

```diff
@@ -1,14 +1,41 @@
+/* eslint-env node */
 const axios = require('./axios');
-const pulls = require('./pulls.js');
+const axiosGH = require('axios');
 const updatesUrl = 'http://updates.jenkins.io/plugin-documentation-urls.json';
 const installsUrl = 'https://stats.jenkins.io/jenkins-stats/svg/' + lastReportDate() + '-plugins.csv';
 const httpCache = {};
+const gitHubToken = process.env.GITHUB_TOKEN;
+const graphql = 'query {'+
+ ' organization(login:"jenkinsci") {'+
+ '   project(number:3 ) {'+
+ '     columns (first:100) {'+
```

Comment:
This isn't going to paginate, so no more than like 100 entries I think?

Reply:
IIUC this is limiting number of columns, which is curently 4 and won't grow much. Number of fetched cards per column is not limited (for some reason GH enforces `first` for `columns` but not `cards` ...)

Reply:
Your so right. Thats what my notes have as well (I assume you saw https://gist.github.com/halkeye/4220bc8b0d204af4815d69dd5c9db204)

Reply:
Your gist is where I got the GraphQL from :)